### PR TITLE
#341 - Changed target version to Java 8

### LINKED ISSUE #341: Migrate to Java 8
Docker adapter should be compatible with Java 8, so it runs on older JRE versions

### PR DESCRIPTION
**Pull request by olegmoz**
Closes #341 
Changed target version to Java 8

### CODE REVIEW

File: .github/workflows/ci-checks.yml [hunk 1]
```diff
@@ -1,19 +1,13 @@
...
+"on": push
 jobs:
   maven-build:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: [windows-latest, ubuntu-latest]
-        java: [11, 13]
+        os: [ubuntu-latest]
+        java: [1.8]
```
Comment (olenagerasimova) [inline .github/workflows/ci-checks.yml:10]: @olegmoz are you sure we do not want to build and test docker-adapter on other java except for 1.8?

**Review by olenagerasimova — CHANGES_REQUESTED** [review]
@olegmoz one question, please, take a look

File: .github/workflows/ci-checks.yml (hunk 1 above)
Comment (olegmoz) [reply .github/workflows/ci-checks.yml]: @olenagerasimova thank you very much for noticing! I've reverted all unnecessary changes in `ci-checks.yml`, now only `1.8` version added

**Review by olenagerasimova — APPROVED** [review]
@olegmoz thanks, good to merge

**Comment by olegmoz** [comment]
@olenagerasimova I had to fix code style issue, please re-approve the PR

**Review by olenagerasimova — APPROVED** [review]

**Pull request merged**